incubator/binding-risingwave.spec/src/main/scripts/config/proxy.yaml

```diff
@@ -0,0 +1,43 @@
+]#
+# Copyright 2021-2023 Aklivity Inc
+#
+# Licensed under the Aklivity Community License (the "License"); you may not use
+# this file except in compliance with the License.  You may obtain a copy of the
+# License at
+#
+#   https://www.aklivity.io/aklivity-community-license/
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+---
+name: test
+catalogs:
+  catalog0:
+    type: test
+    options:
+      url: http://localhost:8081
+
+bindings:
+    app0:
```

Comment:
Proxy configs are typically `app0` to `app1` if the protocol is high level and same on both sides.